core/server/common/src/main/java/alluxio/Process.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+/**
+ * Interface representing an Alluxio process.
+ */
+public interface Process {
+
+  /**
+   * Starts the Alluxio process. This call may block until the process is stopped via
```

Comment:
Does it always block? The difference between a synchronous and asynchronous call seems like it would affect the usage of this method significantly (ie. how it is used in `ProcessUtils`)